packages/portal/src/plugins/error.js

```diff
@@ -23,27 +16,40 @@ function normaliseErrorWithCode(errorOrCode, { scope } = {}) {
 
   if (typeof errorOrCode === 'object') {
     error = errorOrCode;
-    if (SCOPED_CODES[scope]?.[error.statusCode]) {
-      error.code = SCOPED_CODES[scope][error.statusCode];
-    } else if (GENERIC_CODES[error.statusCode]) {
-      error.code = GENERIC_CODES[error.statusCode];
+    if (HTTP_CODES[error.statusCode]) {
+      const httpCode = HTTP_CODES[error.statusCode]; // || 'UnknownError';
+      error.code = `${scope}${httpCode}`;
     }
   } else {
-    error = new Error(this.$i18n.t(`errorMessage.${errorOrCode}.title`));
+    error = new Error;
     error.code = errorOrCode;
   }
 
-  // TODO: consider if we should do this
-  // if (!error.code) {
-  //   error.code = 'unknown';
+  return error;
+}
+
+function translateErrorWithCode(error, { tValues = {} }) {
+  // if (!this.$i18n.te(`errorMessage.${error.code}`)) {
+  //   error.code = 'genericUnknownError';
   // }
 
+  for (const tKey in (this.$i18n.t(`errorMessage.${error.code}`) || {})) {
+    const tValuesForKey = tValues[tKey] || {};
+    tValuesForKey.newline = '<br />';
+    error[tKey] = this.$i18n.t(`errorMessage.${error.code}.${tKey}`, tValuesForKey);
+  }
+
+  if ((error.message === '') && error.title) {
+    error.message = error.title;
+  }
+
   return error;
 }
 
 // TODO: APM captureError?
-export function handleError(errorOrCode, { scope } = {}) {
-  const error = normaliseErrorWithCode.bind(this)(errorOrCode, { scope });
+export function handleError(errorOrCode, options = {}) {
+  let error = normaliseErrorWithCode.bind(this)(errorOrCode, options);
```

Comment:
with the  translation split out, there's no need for bind on the normalise method.